src/lib/SwapRouter/Dexes/Dexter.js

```diff
@@ -0,0 +1,80 @@
+const {
+  percentToDecimal,
+  isTez,
+  isToken,
+  secondsFromNow,
+  convertToMuTez,
+  fromOpOpts,
+  isValidDexFee,
+} = require("../utils.js");
+const { getAmmSwapOutput } = require("../SwapRates/amm");
+const { addTokenApprovalOperators } = require("../TokenTypes");
+const DEX_FEE = 0.1;
+const BURN_FEE = 0.1;
```

Comment:
Dexter has no burn fee, this was added to liquidity baking. Dex fee in dexter is 0.3% as can be seen in the contract code here:  https://gitlab.com/camlcase-dev/dexter/-/blob/master/ligo/dexter.ligo?ref_type=heads#L432

Reply:
Resolved